ballerina/tests/constraint_tests.bal

```diff
@@ -0,0 +1,290 @@
+}
+service object {
+    remote function onRequest(StringConstraintMessage msg) returns string {
+        log:printInfo("Message Received: " + msg.toString());
+        receivedValidStringValue = msg.content;
+        return "Hello Back!";
+    }
+};
+
+Service maxLengthStringService =
+@ServiceConfig {
+    subject: maxLengthStringSubject
+}
+service object {
+    remote function onMessage(StringConstraintMessage msg) {
+        log:printInfo("Message Received: " + msg.toString());
+    }
+
+    remote function onError(Message message, Error err) {
+        log:printInfo("Error Received: " + err.message());
+        if err is PayloadValidationError {
+            receivedStringMaxLengthConstraintError = err.message();
+        }
+    }
+};
+
+Service maxValueIntService =
+@ServiceConfig {
+    subject: maxValueIntSubject
+}
+service object {
+    remote function onMessage(IntConstraintMessage msg) {
+        log:printInfo("Message Received: " + msg.toString());
+    }
+
+    remote function onError(Message message, Error err) {
+        log:printInfo("Error Received: " + err.message());
+        if err is PayloadValidationError {
+            receivedIntMaxValueConstraintError = err.message();
+        }
+    }
+};
+
+Service minValueFloatService =
+@ServiceConfig {
+    subject: minValueFloatSubject
+}
+service object {
+    remote function onMessage(Price msg) {
+        log:printInfo("Message Received: " + msg.toString());
+    }
+
+    remote function onError(Message message, Error err) {
+        log:printInfo("Error Received: " + err.message());
+        if err is PayloadValidationError {
+            receivedFloatMinValueConstraintError = err.message();
+        }
+    }
+};
+
+Service maxValueNumberService =
+@ServiceConfig {
+    subject: maxValueNumberSubject
+}
+service object {
+    remote function onMessage(Weight msg) {
+        log:printInfo("Message Received: " + msg.toString());
+    }
+
+    remote function onError(Message message, Error err) {
+        log:printInfo("Error Received: " + err.message());
+        if err is PayloadValidationError {
+            receivedNumberMaxValueConstraintError = err.message();
+        }
+    }
+};
+
+Service maxLengthArrayService =
+@ServiceConfig {
+    subject: maxLengthArraySubject
+}
+service object {
+    remote function onMessage(NameList msg) {
+        log:printInfo("Message Received: " + msg.toString());
+    }
+
+    remote function onError(Message message, Error err) {
+        log:printInfo("Error Received: " + err.message());
+        if err is PayloadValidationError {
+            receivedArrayMaxLengthConstraintError = err.message();
+        }
+    }
+};
+
+Service validRecordService =
+@ServiceConfig {
+    subject: validRecordSubject
+}
+service object {
+    remote function onRequest(Child msg) returns string {
+        log:printInfo("Message Received: " + msg.toString());
+        receivedValidRecordValue = msg;
+        return "Hello Back!";
+    }
+};
```

Comment:
Do we have any test case by disabling the config?

Reply:
ADded in https://github.com/ballerina-platform/module-ballerinax-nats/pull/483/commits/ebdccbfe524b83c45f4a5bc28e2471df47276366